File: expectanim/src/main/java/com/github/florent37/expectanim/core/scale/ScaleAnimExpectationOriginalScale.java
package com.github.florent37.expectanim.core.scale;

import android.support.annotation.Nullable;
import android.view.View;

/**
 * Created by florentchampigny on 17/02/2017.
 */
public class ScaleAnimExpectationOriginalScale extends ScaleAnimExpectation {

    public ScaleAnimExpectationOriginalScale() {
        super(null, null);
    }

    @Override
    public Float getCalculatedValueScaleX(View viewToMove) {
        return 1f;
    }

    @Override
    public Float getCalculatedValueScaleY(View viewToMove) {
        return 1f;
    }
}

File: expectanim/src/main/java/com/github/florent37/expectanim/core/scale/ScaleAnimExpectationOriginalScale.kt
package com.github.florent37.expectanim.core.scale

import android.view.View

/**
 * Created by florentchampigny on 17/02/2017.
 */
class ScaleAnimExpectationOriginalScale : ScaleAnimExpectation(null, null) {

    override fun getCalculatedValueScaleX(viewToMove: View): Float? {
        return 1f
    }

    override fun getCalculatedValueScaleY(viewToMove: View): Float? {
        return 1f
    }
}
